src/common/languageColors.json

```diff
@@ -0,0 +1,430 @@
+{
+    "1C Enterprise": "#814CCC",
```

Comment:
Woho nice! where did you get these colors? 🤔 from the GitHub Linguist repo?

Reply:
I got these colors from this repo https://github.com/ozh/github-colors/blob/master/colors.json